system/include/emscripten/bind.h

```diff
@@ -1346,6 +1347,39 @@ namespace emscripten {
     };
 
     ////////////////////////////////////////////////////////////////////////////////
+    // CONTAINERS
```

Comment:
Some nit picks for this block of code: space after commas, space after colons, split the ContainerIterator constructor into multiple lines.